### DISCUSSION TITLE
Species of interest not present in Ensembl database

### DISCUSSION THREAD

**Discussion by ArunMaurya8** [stuart-lab/signac#1671] [Q&A]
Hello, I am doing single cell multiomics (RNA+ATAC) analysis for Xenopus laevis. However, for this species there is no official genome/transcriptome release available in Ensembl's core database, thus I am unable to create an EnsDb object for annotations. Please let me know what can done for this species for ATAC seq part. Also, when trying to create an EnsDb from the gtf file some annotations are missing like gene_biotype etc.

For Xenopus laevis, I found https://bioconductor.org/packages/release/data/annotation/html/org.Xl.eg.db.html,  but not sure if it helps.

**Comment by timoast** [comment]
Please see the FAQ here: https://stuartlab.org/signac/articles/faq#an-annotation-or-genome-sequence-for-my-organism-is-not-available-on-bioconductor-what-do-i-do
